modules/virt-attaching-vm-secondary-network-cli.adoc

```diff
@@ -45,7 +45,7 @@ spec:
 ----
 <1> The name of the bridge interface.
 <2> The name of the network. This value must match the `name` value of the corresponding `spec.template.spec.domain.devices.interfaces` entry.
-<3> The name of the network attachment definition, prefixed by the namespace where it exists. The namespace must be either the `default` namespace or the same namespace where the VM is to be created.
+<3> The name of the network attachment definition, prefixed by the namespace where it exists. The namespace must be either the `default` namespace or the same namespace where the VM is to be created. Note that in this case `multus` is used. Multus is a "meta" cloud network interface (CNI) plug-in that allows multiple CNIs to exist so that a pod or virtual machine can use the interfaces it needs.
```

Comment:
Your peer reviewer will probably tell you to use the note callout for the multus sentences.

"Note that in this case". That raises a question -- is multus an option or is it mandatory? If it is an option, then you can say, "In this example, multus is used." If it is mandatory, you should remove "Note that in this case..." and just define multus.